sql-cli/include/base/config/default/configuration.yml

```diff
@@ -3,7 +3,7 @@ connections:
 ## Example of SQLite connection
   - conn_id: sqlite_conn
     conn_type: sqlite
-    host: ../../data/imdb.db
+    host: ../../data/movies.db
```

Comment:
Since a Sqlite file represents a database and can contain multiple tables, I don't think the database file should match the table name used. An alternative could be to have the SQL query referencing `imdb.movies` (SQL Lite database . SQLite tale)